smartseq2_single_sample/tasks/picard.wdl

```diff
@@ -4,38 +4,84 @@ task CollectMultipleMetrics {
   String output_filename
   
   command {
-    java -Xmx4g -jar picard.jar CollectMultipleMetrics \
+    java -Xmx6g -jar /usr/picard/picard.jar CollectMultipleMetrics \
       VALIDATION_STRINGENCY=SILENT \
       METRIC_ACCUMULATION_LEVEL=ALL_READS \
-      INPUT=${aligned_bam} \
+      INPUT="${aligned_bam}" \
       OUTPUT="${output_filename}" \
-      FILE_EXTENSION = ".txt" \
-      PROGRAM = CollectAlignmentSummaryMetrics \
-      PROGRAM = CollectInsertSizeMetrics \
-      PROGRAM = CollectGcBiasMetrics \
-      PROGRAM = RnaSeqMetrics \
-      REFERENCE_SEQUENCE=${ref_genome_fasta} \
+      FILE_EXTENSION=".txt" \
+      PROGRAM=null \
```

Comment:
What does this do? I never quite understood how picard uses this.

Reply:
I just put it here to avoid further unnecessary error from picards